.github/workflows/php-tests.yml

```diff
@@ -10,6 +10,9 @@ on:
         required: false
         default: true
         type: boolean
+      phpunit-config:
```

Comment:
Would a `-file` being prepended be more clear what exactly this input is for - so `phpunit-config-file`?